lib/cache_crispies/base.rb

```diff
@@ -241,13 +251,14 @@ def self.serialize(
     )
       attribute_names.flat_map do |attrib|
         attrib = attrib&.to_sym
+        key = attrib
         current_nesting = Array(@nesting).dup
         current_conditions = Array(@conditions).dup
 
         @attributes <<
           Attribute.new(
-            attrib,
-            from: from,
+            transform_key(attrib),
+            from: from || attrib,
```

Comment:
Why is the `|| attrib` needed here?

Reply:
I'm not great at seeing GH notifications, sorry.
I plan to check out your comments here and address in the next week or two.

I'll need to get back in the code here to recall correctly, but I believe this was meant as an improvement.
It seems like it calls for adding a comment to explain why, lol